generator/source/contact.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: page
+title: Contact
+permalink: /contact/
+feature-img: "../examples/img/hero-bond-bg.jpg"
+---
+
+# Contributing to Hyperledger Sawtooth
+
+Hyperledger Sawtooth is Apache 2.0 licensed and accepts contributions via
+[GitHub](https://github.com/hyperledger/sawtooth-core) pull requests.
+
+Please see [Contributing](https://sawtooth.hyperledger.org/docs/core/releases/la
```

Comment:
We could move the community page to the website as well, since it is not release specific (or sawtooth-core specific).